pandas/core/series.py

```diff
@@ -2016,10 +2016,124 @@ def _try_kind_sort(arr):
         else:
             return result.__finalize__(self)
 
-    @Appender(generic._shared_docs['sort_index'] % _shared_doc_kwargs)
     def sort_index(self, axis=0, level=None, ascending=True, inplace=False,
                    kind='quicksort', na_position='last', sort_remaining=True):
+        """
+        Sort object by labels.
+
+        Returns a new Series sorted by label if `inplace` argument is `False`,
+        otherwise updates the original series and returns `null`.
+
+        Parameters
+        ----------
+        axis : int, default 0
+            Axis to direct sorting.
+        level : int, default None
+            If not None, sort on values in specified index level(s).
+        ascending : boolean, default true
+            Sort ascending vs. descending.
+        inplace : bool, default False
+            If True, perform operation in-place.
+        kind : {'quicksort', 'mergesort', 'heapsort'}, default 'quicksort'
+            Choice of sorting algorithm. See also ndarray.np.sort for more
+            information.  `mergesort` is the only stable algorithm. For
```

Comment:
you can use normal quotes in this case (like `'mergesort'`) to indicate it is a string